doc/source/user_guide/meshing_workflow/new_meshing_workflows.rst

```diff
@@ -0,0 +1,701 @@
+Use the **Watertight Geometry** workflow for water-tight CAD geometries that
+do not require much in the way of clean-up or modifications.
+The following example shows you how to use the Watertight Geometry workflow.
+
+Import geometry
+~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    import ansys.fluent.core as pyfluent
+    from ansys.fluent.core import examples
+
+    import_file_name = examples.download_file('mixing_elbow.pmdb', 'pyfluent/mixing_elbow')
+    meshing = pyfluent.launch_fluent(
+        mode="meshing", precision='double', processor_count=2
+    )
+    watertight = meshing.watertight()
+    watertight.import_geometry.file_name.set_state(import_file_name)
+    watertight.import_geometry.length_unit.set_state('in')
+    watertight.import_geometry()
+
+Add local sizing
+~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.add_local_sizing.add_child_to_task()
+    watertight.add_local_sizing()
+
+Generate surface mesh
+~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.create_surface_mesh.cfd_surface_mesh_controls.max_size.set_state(0.3)
+    watertight.create_surface_mesh()
+
+Describe geometry
+~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.describe_geometry.update_child_tasks(setup_type_changed=False)
+    watertight.describe_geometry.setup_type.set_state("The geometry consists of only fluid regions with no voids")
+    watertight.describe_geometry.update_child_tasks(setup_type_changed=True)
+    watertight.describe_geometry()
+
+Update boundaries
+~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.update_boundaries.boundary_label_list.set_state(["wall-inlet"])
+    watertight.update_boundaries.boundary_label_type_list.set_state(["wall"])
+    watertight.update_boundaries.old_boundary_label_list.set_state(["wall-inlet"])
+    watertight.update_boundaries.old_boundary_label_type_list.set_state(["velocity-inlet"])
+    watertight.update_boundaries()
+
+Update regions
+~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.update_regions()
+
+Add boundary layers
+~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.add_boundary_layer.add_child_to_task()
+    watertight.add_boundary_layer.insert_compound_child_task()
+    watertight.task("smooth-transition_1").bl_control_name.set_state("smooth-transition_1")
+    watertight.add_boundary_layer.arguments = {}
+    watertight.task("smooth-transition_1")()
+
+Generate volume mesh
+~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    watertight.create_volume_mesh.volume_fill.set_state("poly-hexcore")
+    watertight.create_volume_mesh.volume_fill_controls.hex_max_cell_length.set_state(0.3)
+    watertight.create_volume_mesh()
+
+Switch to solution mode
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+    solver = meshing.switch_to_solver()
+
+Fault-tolerant meshing workflow
+-------------------------------
+Use the **Fault-tolerant** meshing workflow for more complicated non-water-tight CAD
+geometries that may require some form of clean-up or modification (for example,
```

Comment:
```suggestion
geometries that might require some form of clean-up or modification (for example,
```
"May" should be used only when related to permissions, especially in regard to legal permissions.